### PR TITLE
add ledger_id to info requests

### PR DESCRIPTION
**Pull request by SimiHunjan**
Signed-off-by: SimiHunjan <simi.hunjan.k@gmail.com>

**Description**:

Add ledger ID HIP

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: HIP/hip-0000-add-ledger-id-field.md [hunk 1]
```diff
@@ -0,0 +1,575 @@
+- hip: <HIP number (this is determined by the HIP editor)>
+- title: Add Network ID Field to Info Responses
+- author: Simi Hunjan <simi@hedera.com>, David Matusevich <davidmatusevich@swirlds.com>
+- type: Standards
+- category: Services
+- needs-council-approval: true
+- status: Draft
+- created:
```
Comment (sergmetelin) [inline HIP/hip-0000-add-ledger-id-field.md:8]: Do you mind adding the date 11-2-2021?

File: HIP/hip-0000-add-ledger-id-field.md [hunk 2]
```diff
@@ -0,0 +1,575 @@
+- hip: <HIP number (this is determined by the HIP editor)>
```
Comment (sergmetelin) [inline HIP/hip-0000-add-ledger-id-field.md:1]: Do you mind assigning hip-33 to this HIP? And rename the file

File: HIP/hip-0000-add-ledger-id-field.md [hunk 3]
```diff
@@ -0,0 +1,575 @@
...
+- title: Add Network ID Field to Info Responses
+- author: Simi Hunjan <simi@hedera.com>, David Matusevich <davidmatusevich@swirlds.com>
+- type: Standards
+- category: Services
```
Comment (sergmetelin) [inline HIP/hip-0000-add-ledger-id-field.md:5]: Following the rest of the naming, let's keep it singular rather than plural - `Service`

**Review by sergmetelin — CHANGES_REQUESTED** [review]

**Comment by SimiHunjan** [comment]
deleted branch so submitted a new PR with comment fixes https://github.com/hashgraph/hedera-improvement-proposal/pull/191

**Pull request closed without merging**